api/v2.0/swagger.yaml

```diff
@@ -1951,6 +1951,259 @@ paths:
           $ref: '#/responses/403'
         '500':
           $ref: '#/responses/500'
+  '/projects/{project_name_or_id}/webhook/policies':
+    get:
+      summary: List project webhook policies.
+      description: |
+        This endpoint returns webhook policies of a project.
+      tags:
+        - webhook
+      operationId: ListWebhookPolicy
```

Comment:
Suggest to rename `ListWebhookPolicy` to `listWebhookPoliciesOfProject` and change the tag to `project` so that when we want to introduce the system level webhook we can let the `operationId` for the system level be `listWebhookPolicies` and tag it to `webhook`.

Reply:
update the names, but let's keep the tag as webhook till we introduce the system level webhook